CompanyEmployees.Core.Domain/Repositories/IAttendanceRepository.cs

```diff
@@ -0,0 +1,13 @@
+using CompanyEmployees.Core.Domain.Entities;
+
+namespace CompanyEmployees.Core.Domain.Repositories;
+
+public interface IAttendanceRepository
+{
+    Task<IEnumerable<Attendance>> GetAllAttendancesAsync(bool trackChanges, CancellationToken ct = default);
+    Task<Attendance> GetAttendanceAsync(Guid attendanceId, bool trackChanges, CancellationToken ct = default);
+    Task<IEnumerable<Attendance>> GetAttendancesByEmployeeIdAsync(Guid employeeId, bool trackChanges, CancellationToken ct = default);
+    Task<Attendance?> GetTodayAttendanceByEmployeeIdAsync(Guid employeeId, bool trackChanges, CancellationToken ct = default);
```

Comment:
You're using a nullable return type for GetTodayAttendanceByEmployeeIdAsync, which is good for conveying absence of data. Ensure consistency by doing the same for GetAttendanceAsync, which may also return null if the attendance ID does not exist.

**Suggested change**
```suggestion
Task<Attendance?> GetAttendanceAsync(Guid attendanceId, bool trackChanges, CancellationToken ct = default);
```